src/components/base/misc/Icon.svelte

```diff
@@ -1,51 +1,4 @@
 <script context="module" lang="ts">
-	const featherIcons = [
```

Comment:
I'm migrating to a lazy-loaded icon library called iconify. I intend to eventually let the user bring their own icon pack, so these are strictly used as default icons for some component in this project